### PR TITLE
Update docs for pnp ios version 8.1.0 & android 7.1.1

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

**Review by shahbaz17 — APPROVED** [review]
LGTM

Merge it when the release is out.

**Review by shahbaz17 — APPROVED** [review]
lgtm

File: docs/sdk/pnp/ios/usage.mdx [hunk 1]
```diff
@@ -74,12 +75,13 @@ public struct W3ALoginParams: Codable {
...
 
-Use getPrivkey() to get the private key of the user. The method returns an EVM compatible private key which can be used to sign transactions on EVM
-compatible chains.
+Use getPrivkey() to get the private key of the user. The method returns an EVM compatible private
+key which can be used to sign transactions on EVM compatible chains.
 
 ### `getEd25519PrivKey()`
 
-Use getEd25519PrivKey() to get the Ed25519 private key of the user. This private key can be used to sign transactions on Solana.
+Use getEd25519PrivKey() to get the Ed25519 private key of the user. This private key can be used to
```
Comment (will8683) [inline docs/sdk/pnp/ios/usage.mdx:83]: False

**Review by will8683 — COMMENTED** [review]
No it's not helping me only u

**Pull request merged**